src/R/Components/Test/Plots/RPlotIntegrationTest.cs

```diff
@@ -25,6 +25,8 @@
 
 namespace Microsoft.R.Components.Test.Plots {
     [ExcludeFromCodeCoverage]
+    [Category.Plots]
+    [Collection(CollectionNames.NonParallel)]
```

Comment:
The main reason to use `NonParallel` is lack of isolated context. For example, shared files that can be overwritten or deleted or global singletons that carry their state between tests but are expected to be reset to some basic state between tests. `Microsoft.R.Components.Test` shouldn't have such cases cause it has individual MEF containers for every test and AFAIK doesn't use shared external resources that can be changed by several tests.